mail_compose_message_reply_to/models/mail_compose_message.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailComposeMessageReplyTo(models.TransientModel):
+    _inherit = "mail.compose.message"
+
+    @api.multi
+    def get_mail_values(self, res_ids):
+        self.ensure_one()
+        results = super(MailComposeMessageReplyTo, self).get_mail_values(res_ids)
+        for res_id in res_ids:
+            if (
+                self.reply_to
+                and "reply_to" not in results[res_id].keys()
+                and "@" not in str(results[res_id]["reply_to"])
```

Comment:
@tack-yabeQuartile I don't think this check is necessary here.  If anything, the format check should be done on the reply_to field in the template.

But in case you do need to validate the format, here is what we have done in the past for your reference: https://github.com/qrtl/nrq-custom/blob/5bc5dc12f4b8779d5c1dd4220bb1d53d241b406c/l10n_jp_hr_employee/models/hr_private_info.py#L445-L454

Reply:
@yostashiro Thanks for the comment!
The link was very helpful, so I remember. For now, I choose not to do excess judgement.